include/php-sql-parser.php

```diff
@@ -2059,7 +2059,7 @@ private function load_reserved_words()
             'zerofill'
             );
 
-        for ($i=0;$i<count($this->reserved);++$i) {
+        for ($i=0, $iMax = count($this->reserved); $i< $iMax; ++$i) {
```

Comment:
This doesn't really matter, but some of these are formatted as `$i< $iMax` and some are formatted as `$i < $iMax`. I'm surprised the linter doesn't enforce the latter.

Reply:
Thanks! I had assumed that the linter would have taken care of the formatting. Will update this PR.